README.md

```diff
@@ -77,14 +92,12 @@ To begin, an `App ID` for the project must be created in the iOS Developer Membe
 #### 2. Login to Team in Xcode
 Once an App ID is set up in the developer centre, navigate to XCode and make sure you are signed into your Team's Apple ID. You can check if you have already done this by navigating to Xcode and selecting `Xcode -> Preferenes -> Accounts` and checking if your team appearing under the subheading "Apple IDs". if it doesn't, add it, by selecting the "+" button and entering the authentication details.
 
-#### 3. Create Private Certificates Repository
-You will now need to set up a private git repository, which is where your code signing documents will be securely kept. Navigate to the site where you wish to host this repository (e.g. github, bitbucket) and create an empty repository. The recommended name is "{PROJECT_NAME}-Certificates".
+#### 3. Create/Select Private Certificates Repository
+If this is the first app created this client/team, then you will need to set up the certificates repository. This is where your code signing documents will be securely kept. One repository should be used for all apps for a given client, so if an app exists for this client/team re-use the existing one and skip this step. Navigate to the site where you wish to host this repository (github is now the preferred host, not bitbucket, since we have unlimited private repos now) and create an empty repository. The recommended name is "{CLIENT_NAME}-Certificates".
```

Comment:
"created for this ... you will need to create a new certificates repository".

Also it's good to put newlines in Markdown files, makes it easier to review. I use an Atom plugin that does it automatically on Cmd-Alt-Y.